packages/image_picker/image_picker_platform_interface/lib/src/types/multi_image_picker_options.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Specifies options for picking multiple images from the device's gallery.
+class MultiImagePickerOptions {
+  /// Creates an instance with the given [maxHeight], [maxWidth], [imageQuality]
+  /// and [requestFullMetadata].
+  const MultiImagePickerOptions({
+    this.maxHeight,
+    this.maxWidth,
+    this.imageQuality,
+    this.requestFullMetadata = true,
+  });
+
+  /// The maximum width of the image, in pixels.
+  ///
+  /// If null, the image will only be resized if [maxHeight] is specified.
+  final double? maxWidth;
+
+  /// The maximum height of the image, in pixels.
+  ///
+  /// If null, the image will only be resized if [maxWidth] is specified.
+  final double? maxHeight;
+
+  /// Modifies the quality of the image, ranging from 0-100 where 100 is the
+  /// original/max quality.
+  ///
+  /// Compression is only supported for certain image types such as JPEG. If
+  /// compression is not supported for the image that is picked, a warning
+  /// message will be logged.
+  ///
+  /// If null, the image will be returned with the original quality.
+  final int? imageQuality;
```

Comment:
Since 100 means the original quality, we can make this non-null where the default is 100

Reply:
If we do we'll need to audit the native implementations, since I believe the existing codepaths just check for null to decide whether to bypass the extra steps.

Reply:
OK, that makes sense. Let's keep it as nullable.